diagrams/tutorials/web/css-rule-explained.drawio

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2021-10-18T04:05:36.192Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/95.0.4638.49 Safari/537.36" etag="m4_M-DRE7O73LJxVvYYD" version="15.5.4" type="device"><diagram id="OH_mBuzn8ZVgwBRywq3m" name="Page-1">7Vhdc9soFP01nj45IwnJH4+24my7nc50omnT7BuRsMQEgRbh2Omv70VCsj6TNk48O5l90cAFLnDP4XDFBPnp4S+Js+SLiAibOFZ0mKDLieMsHBu+2vBYGuYzY4gljUpTwxDQn8QYLWPd0YjkrY5KCKZo1jaGgnMSqpYNSyn27W5bwdqzZjgmPUMQYta33tBIJWZbnnW0fyQ0TqqZbcu0pLjqbAx5giOxb5jQZoJ8KYQqS+nBJ0zHropLOe5qpLVemCRc/c4AJG8xDT7zWHEWfrv+8W3u3kznpZcHzHZmw2ax6rGKgBQ7HhHtxJqg9T6higQZDnXrHiAHW6JSBjUbilvB1RVOKdNwX4s7oYSxBmIni0GJUoCd46EVfGC1+qM75BexEDEjOKP5RSjSoiHMi65X29InFDteS8LYLtRzJcV9jZKje1DGfMGELHaCNvZ6s5mDHTMac7CFEDoi66FVVy44OF2b0BCpyGE05naNJJwAIlKiJCzTMgNctCiHGPYj19T3Ry4tTciTBo0qzmDD3rj2fAQYCgbjP8Dbfdd4V7gyslV9+H3PX22Wb4l2pQoV2jOvh7ZT9WnCvfBOh3v16dPuZhMEKPHJnn/8Z+P9/XXqOQN4z5gOTkQfoBirYuvommbfMb+h/B5oAOybr6t+MG+r68DoGU41LfhdnhV1SyM1zQuoAHQLYnhojH2xR0OKwueHkhYfJo6v6Yt5Ps2JpNsT5wHw1dTQSE9TK8QpTlMsY8qnSmSvFo87HN7HxUmdhiWPteOAYUXWBdQneW+4vAWW6VvrD/zNL8f6jiqN/TKlcawvgo8Ijl49iM2I1BRCU8vM0eG6ctiQDiMQHXHJYaWUx2BYQC0D7sGRJTKozdbTStOTlQHxeeJe6SiNa/WVZjGgNDZ6q5vFHlCaDt6QAWW6GO4ke1xLoDBRzwPfZklf9F9BuNHSa4cT9YUbDUXTfatojuu25nMrrLN/d6JqaIiu7YLI1I3VGfSDANqud1rl6zNauhw5pFoR24AMkvqJA1NrqMaIQmq9Mg0pjSI9zSAD2mnIOfOM1yCU3SHUsn8+h47nm51Or8enSxIyLLGigufvKQccS/rOlvPPrY42W33sl+fEftbDvlSBgDD4Y4bdv1/0z//H53VuZqBDD/36jeAs8C+fv5gJj1b6rUQHiOE8p+HYFTymxU9lYicDD63rpvsGA8iBqh96XReeqd2aNevy5cEsuag8tpKyfgpBot5T0LOoN1Ad+rGrbJJAZk4f2u6HkDYzfBW0uOUrSXHdTn7SIUteRNyMar4BdRzN3GXb0azz56ngV4WonqOCePW2T0gSfyNL/J+M/20yeotObmO9kIzuonNRetYrkRGqx3fVsvvxcRptfgE=</diagram></mxfile>
```

Comment:
Let's use sentence case for labels:

* CSS Rule -> CSS rule
* CSS Selector -> CSS selector
* (Declarations is good as-is)